parse/parse.go

```diff
@@ -397,7 +392,7 @@ func (p *parser) parseExpression() (Expression, error) {
 	case itemFunction, itemPrivateStart, itemReservedStart:
 		p.backup()
 	case itemExpressionClose: // empty expression
-		return expr, nil
+		return errorf("%w: missing operand or annotation", mf2.ErrSyntax)
```

Comment:
Duplicate ErrSyntax.

https://github.com/expect-digital/go-mf2/pull/149/files#diff-5ed87adddc43e6af620e98ca4d4a40fe884b5d4b6bf1c9d7f2352f68f34328c4R156 already adds ErrSyntax.

```suggestion
		return errorf("missing operand or annotation")
```